docs/project/2024/ana_maria.balutoiu/index.md

```diff
@@ -0,0 +1,104 @@
+| [Hall Effect Sensor (3144)](https://ardushop.ro/ro/electronica/483-senzor-magnetic-hall-a3144.html?gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pcTJfPkq9zg1-BR94EmOeKQr0mzdFMify2V8sOvuPWqg-Te-TSuL4BoCO6YQAvD_BwE) | Detects the presence of a magnet(door open). | [3.21 RON](https://ardushop.ro/ro/electronica/483-senzor-magnetic-hall-a3144.html?gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pcTJfPkq9zg1-BR94EmOeKQr0mzdFMify2V8sOvuPWqg-Te-TSuL4BoCO6YQAvD_BwE) |
+| [555 Timer IC](https://docs.rs-online.com/ae11/A700000009214226.pdf) | Generates an adjustable alarm tone. | [2.41 RON](https://ardushop.ro/ro/electronica/199-ic-timer-555.html?search_query=NE555&results=7) |
+| [Buzzer](https://www.farnell.com/datasheets/2171929.pdf) | Produces the alarm sound. | [3.33 RON](https://quintrix.ro/buzzer-activ-12v-arduino-raspberry?search=buzzer) |
+| [Transistor (BC547)](https://www.farnell.com/datasheets/410427.pdf) | Controls the 555 timer. | [1.68 RON](https://quintrix.ro/index.php?route=product/product&product_id=8438&search=BC547) |
+| [Reed Switch](https://en.wikipedia.org/wiki/Reed_switch) | Mounted on the door itself. | [1.49 RON](https://www.optimusdigital.ro/en/buttons-and-switches/3899-normally-open-magnetic-reed-switch-2x14-mm.html) |
+| [10uF Capacitor (C1)](https://www.emag.ro/condensator-electrolitic-10uf-50v-dc-105-c-samxon-km-10u-50v-t128611/pd/DDYYNXMBM/?cmpid=101180&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Iluminat_and_electrice&utm_content=76376892625&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pd9ThOIVmEhijwZhV7bZUx1GJTYeP-z-NqkZEjqQCJ8S-3f6G47xSRoCUpoQAvD_BwE) | Connected to pin 2 of the 555 timer. | [4.41 RON](https://www.emag.ro/condensator-electrolitic-10uf-50v-dc-105-c-samxon-km-10u-50v-t128611/pd/DDYYNXMBM/?cmpid=101180&utm_source=google&utm_medium=cpc&utm_campaign=(RO:eMAG!)_3P_NO_SALES_%3e_Iluminat_and_electrice&utm_content=76376892625&gad_source=1&gclid=CjwKCAjw88yxBhBWEiwA7cm6pd9ThOIVmEhijwZhV7bZUx1GJTYeP-z-NqkZEjqQCJ8S-3f6G47xSRoCUpoQAvD_BwE) |
+| [50k Potentiometer (RV1)](https://www.optimusdigital.ro/en/potentiometers/1885-50k-mono-potentiometer.html?search_query=50k&results=72) | Adjusts the alarm frequency. | [1.49 RON](https://www.optimusdigital.ro/en/potentiometers/1885-50k-mono-potentiometer.html?search_query=50k&results=72) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [rppal](https://github.com/golemparts/rppal) | Raspberry Pi Peripheral Access Library | Provides an interface for accessing various Raspberry Pi peripherals |
+| [panic-halt](https://github.com/korken89/panic-halt) | panic handler that halts the program | Used to handle panics gracefully |
+| [elf2uf2](https://github.com/JoNil/elf2uf2-rs) | for flashing onto microcontrollers | Converts Rust executables (in ELF format) to UF2 format |
```

Comment:
This is not a library.

Reply:
Thank you! I deleted the elf2uf2 tool from the list